garage/misc/wrappers.py

```diff
@@ -0,0 +1,88 @@
+"""Gym env wrappers"""
```

Comment:
Please do not put these in `misc`, which we are trying to clean out. I think it should go under `garage.envs`.